types/operator.go

```diff
@@ -108,3 +109,10 @@ func BitmapToQuorumIds(bitmap *big.Int) []QuorumNum {
 	}
 	return quorumIds
 }
+
+type QuorumAvsState struct {
```

Comment:
I think instead of loading one file with all the struct we should create new files to keep it cleaner. This is in the `operator` but seems it has relations AVS state?  This file is already getting too big for operator types.

Reply:
I know... we need a proper cleanup of types as we've discussed a few times. Can we do this in another ticket? I need this merged to be able to release inc-sq

Reply:
i have created an issue #42